docs/overview.adoc

```diff
@@ -19,25 +19,141 @@ For example using Kroxylicious proxy it's easier to experiment with alternative
+reverse order, each having the opportunity to inspect and/or manipulating the response. Eventually a response is
+returned to the client.
+
+[a2s, format="svg"]
+....
+.----------------------------------------------------------------------------------------------------------------------.
+|                                                                                                                      |
+|                       '---------------------------------------------------------------'                              |
+|                       |[Kroxylicious]                                                 |                              |
+|                       |                                                               |                              |
+|                       |   '----------------------------------------------------'      |      '--------------------'  |
+|                       |   |[Virtual Cluster]                                   |      |      |[Cluster]           |  |
+|  '-------------'      |   |   '----------'     '----------'     '----------'   |      |      |    '------------'  |  |
+|  |[Client]     |      |   |   |[Filter1] |     |[Filter2] |     |[Filter3] |   |      |      |    |[Broker]    |  |  |
+|  |             |======|===|==>|          |====>|          |====>|          |===|======|======|===>|            |  |  |
+|  |             |  A   |   |   | F(A)-->B |  B  | F(B)-->C |  C  | F(C)-->D |   |      |      | D  |            |  |  |
+|  |             |      |   |   |          |     |          |     |          |   |      |      |    |            |  |  |
+|  |             |<=====|===|===|          |<====|          |<====|          |<==|======|======|====|            |  |  |
+|  |             |  W   |   |   | f(X)-->W |  X  | f(Y)-->X |  Y  | f(Z)-->Y |   |      |      | Z  |            |  |  |
+|  '-------------'      |   |   '----------'     '----------'     '----------'   |      |      |    '------------'  |  |
+|                       |   |                                                    |      |      '--------------------'  |
+|                       |   '----------------------------------------------------'      |                              |
+|                       |                                                               |                              |
+|                       '---------------------------------------------------------------'                              |
+|                                                                                                                      |
+.----------------------------------------------------------------------------------------------------------------------.
+[0,0]: {"fill":"#99d","a2s:delref":1}
+....
+
+As mentioned above, Kroxylicious takes the responsibility to rewrite the Kafka RPC responses that carry broker address
```

Comment:
> I am probably being blind but I'm not seeing the previous reference.

I am referring to this text in the Virtual CLuster section:

>> Kroxylicious automatically intercepts all the Kafka RPC responses that contain a broker address.  It rewrites the address so that it refers to the corresponding broker endpoint of the Virtual Cluster.  This means when the Kafka Client goes to connect to, say broker 0, it does so through the Virtual Cluster.


> 
> I do think there is an important point to make the proxying is entirely transparent to both upstream and downstream sides this is implicit in this section but I think its worth calling out explicitly.

I'll add a sentence to make this clear.